### PR TITLE
Swaggerify /rooms/:room_id/leave

### PR DESCRIPTION
**Pull request by illicitonion**

### CODE REVIEW

**Comment by Kegsay** [comment]
Please don't lie on the name. `{{ joining_http_api }}` contains `/invite` endpoints in the YAML. Probably worth splitting that out too.

File: api/client-server/v1/leaving.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        they were previously allowed to see.
+      security:
+        - accessToken: []
+      parameters:
+        - in: path
+          type: string
+          name: roomId
+          description: The room identifier or room alias to leave.
+          required: true
+          x-example: "#monkeys:matrix.org"
```
Comment (Kegsay) [inline api/client-server/v1/leaving.yaml]: This is not a room ID.

**Comment by Kegsay** [comment]
LGTM other than comments.

File: api/client-server/v1/leaving.yaml (hunk 1 above)
Comment (illicitonion) [inline api/client-server/v1/leaving.yaml]: Done

**Pull request merged**